Educator Resources/Azure Guides/MobileDevelopment.md

```diff
@@ -1,150 +0,0 @@
-![](media/image1.png)
-
-# Mobile Development
```

Comment:
Any way we can update this rather than delete?

Reply:
Not really - the Azure Mobile services are going away and we don't have a decent replacement...

Reply:
Gotcha. Ok, sounds good!